airflow/cli/cli_parser.py

```diff
@@ -434,6 +434,12 @@ def __init__(self, flags=None, help=None, action=None, default=None, nargs=None,
 ARG_CFG_PATH = Arg(
     ("--cfg-path",),
     help="Path to config file to use instead of airflow.cfg")
+ARG_MIGRATION_TIMEOUT = Arg(
+    ("-t", "--migration-wait-timeout"),
+    help="timeout to wait for db to migrate ",
+    type=int,
+    default="0",
```

Comment:
Given the default is 0 (i.e. don't wait) I think we should rename this command,

Rather than `airflow db wait-for-migrations` how about `airflow db check-migrations`, `airflow db is-fully-migrated`? Not 100% happy with either of those so if you have any better ideas?

Reply:
We already have `airflow db check`, so perhaps `airflow db check-migrations`?

Reply:
I think check-migrations is fine.